src/components/Skill/Skill.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Box as MuiBox, BoxProps as MuiBoxProps } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+type TLevelRange = 0 | 1 | 2 | 3 | 4 | 5;
+
+export interface SkillProps extends MuiBoxProps {
+    label?: string
+    level?: TLevelRange;
+}
+
+
+const useStyles = makeStyles(theme => {
+    const { palette } = theme;
+
+    return {
+        root: {
```

Comment:
It could be better to make the skill width dynamic as you need it to be smaller on mobile devices.